### PR TITLE
add Constant, Zeros and Ones

### PR DESCRIPTION
**Pull request by larcai**
similair to the python versions
W = tf.Variable(tf.zeros([784, 10]))
b = tf.Variable(tf.zeros([10]))

### CODE REVIEW

**Comment by migueldeicaza** [comment]
Thanks for the contribution!   I was on vacation, but will review this week as I get back.

File: SampleTest/SampleTest.cs [hunk 1]
```diff
@@ -305,14 +328,26 @@ void LinearRegression ()
...
+            //find _pywrap_tensorflow_internal.pyd in the tensorflow_gpu-1.1.0-cp36-cp36m-win_amd64.whl
+            //the location in the whl is here: \tensorflow_gpu-1.1.0.data\purelib\tensorflow\python\
+            //extract, copy and rename _pywrap_tensorflow_internal.pyd to libtensorflow.dll
+            //copy to you TF# folder
+            //if running with libtensorflow.dll where Python 3.5 environment is needed - append the py35 environment to the path            
+            //see https://github.com/larcai/cognibooks/blob/master/tensorflow/1_tf_setup.workbook 
+            var envpaths = new List<string> { @"C:\ProgramData\Anaconda3\envs\py35" }
+                .Union(Environment.GetEnvironmentVariable("PATH").Split(new char[] { ';' }, StringSplitOptions.RemoveEmptyEntries));
+            Environment.SetEnvironmentVariable("PATH", string.Join(";", envpaths));
+
```
Comment (migueldeicaza) [inline SampleTest/SampleTest.cs]: Why is this needed?   When using the NuGet package, the support library is already included.

Also, I do not want to introduce something with hardcoded paths like that.

**Comment by cesarsouza** [comment]
Hi @larcai, 

Have you got some time to review Miguel's remarks above? Your PR is quite useful, but I think those explicit references to Anaconda are blocking its acceptance. I would also be interested in using Zeros and Ones methods in forthcoming TFS versions 😅 

Regards,
Cesar

**Comment by migueldeicaza** [comment]
I cleaned up the patch by hand and applied it.
